src/lib/dusk.rs

```diff
@@ -0,0 +1,217 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+//
+// Copyright (c) DUSK NETWORK. All rights reserved.
+
+use core::cmp::Ordering;
+use std::fmt;
+use std::num::ParseFloatError;
+use std::ops::{Add, Deref, Div, Mul, Sub};
+use std::str::FromStr;
+
+use rusk_abi::dusk;
+
+/// The underlying unit of Dusk
+pub type Lux = u64;
+
+pub(crate) const MIN: Dusk = Dusk(dusk::LUX);
+pub(crate) const MAX: Dusk = Dusk(dusk::dusk(f64::MAX));
```

Comment:
If here you want to express the maximum *convertible* this is incorrect. That would be something like:
```rust
dusk(f64::MAX / dusk(1.0) as f64)
```